src/Implementation/ColdStorage/ColdStorage/BufferedFrameData.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Practices.IoTJourney.ColdStorage.RollingBlobWriter;
+using Microsoft.ServiceBus.Messaging;
+
+namespace Microsoft.Practices.IoTJourney.ColdStorage
+{
+    public class BufferedFrameData : BlockData
+    {
+        private readonly EventData _lastEventDataInFrame;
```

Comment:
This should stop us from merging but...
`EventData` is disposable. What implications does that have here?